src/gpuarray_buffer_opencl.c

```diff
@@ -811,7 +824,6 @@ static gpukernel *cl_newkernel(gpucontext *c, unsigned int count,
       FAIL(NULL, GA_VALUE_ERROR);
     p = clCreateProgramWithBinary(ctx->ctx, 1, &dev, lengths, (const unsigned char **)strings, NULL, &ctx->err);
     if (ctx->err != CL_SUCCESS) {
-      clReleaseProgram(p);
```

Comment:
Yes, it's an unrelelated bugfix.  If we fail to create the program, we don't need to release it (and it could crash if we aren't lucky).